openapi/kas-fleet-manager.yaml

```diff
@@ -1138,6 +1150,21 @@ components:
         max_connection_attempts_per_sec:
           description: 'Maximium connection attempts per second available to this Kafka instance size.'
           type: integer
+        max_message_size:
+          description: 'Maximum message size available to this Kafka instance size.'
+          type: object
+          $ref: '#/components/schemas/SupportedKafkaSizeBytesValueItem'
+        min_in_sync_replicas:
+          description: "Minimum number of in-sync replicas."
+          type: integer
+        replication_factor:
+          description: "Replication factor available to this Kafka instance size."
+          type: integer
+        supported_az_modes:
+          description: 'List of Availability Zone modes that this Kafka instance size supports. The possible values are "single", "multi".'
+          type: array
+          items:
```

Comment:
PS: The question below might not be applicable as by looking at their description and the validation (corrent??) these values are never going to have a value asked in a question. But thought I could still ask in case of future changes. 

What happens if some of these are empty i.e if an `integer` is `0` and the array is an empty array? 
Is the API expected to send the equivalent values or the desired behaviour is that they'll be omitted from the returned response?

Reply:
This fields are always returned by a GET/LIST endpoint so the user will never set them. The described scenario should never happen. As you mentioned we have validation to ensure that doesn't happen when reading from the configuration file.
In any case, if for some reason they end up with the zero-value whether they will be omitted from the returned response will depend on the marshaling behavior specified in OpenAPI. For example for supported_az_modes if the value of it is `nil` then it would be ommited from the response as the generated data type from OpenAPI is:
```
	SupportedAzModes []string `json:"supported_az_modes,omitempty"`
```
(notice the omitempty).
But as commented, this should never happen in this specific fields (if it does is a bug)

Reply:
Perfect, thanks.